.github/workflows/ci.yml

```
@@ -0,0 +1,32 @@
+name: CI
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)